app/controllers/staff/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Staff::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def azure_activedirectory_v2
+    auth = request.env["omniauth.auth"]
+    email = auth["info"]["email"]
+    azure_ad_uid = auth["azure_ad_uid"]
```

Comment:
```suggestion
    azure_ad_uid = auth["uid"]
```

Is this correct? In the original spike the key we get back from Omniauth is `uid`.

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1463/files#diff-ed2e5712248eb61bb5efb057efaa190974f0925392e36769f873e605a0d167d0R6